framework/PostProcessors/ImportanceRank.py

```diff
@@ -545,26 +361,22 @@ def __runLocal(self, inputDict):
     for target in self.targets:
       featCoeffs = LinearRegression().fit(sampledFeatMatrix, inputDict['targets'][target]).coef_
       featWeights = abs(featCoeffs)/np.sum(abs(featCoeffs))
-      senWeightDict[target] = list(zip(feats,featWeights,self.dimensions))
+      senWeightDict[target] = featWeights
       senCoeffDict[target] = featCoeffs
     for what in self.what:
       if what.lower() == 'sensitivityindex':
         what = 'sensitivityIndex'
-        if what not in outputDict.keys():
-          outputDict[what] = {}
         for target in self.targets:
-          entries = senWeightDict[target]
-          entries.sort(key=lambda x: x[1],reverse=True)
-          outputDict[what][target] = entries
+          for i, feat in enumerate(feats):
+            varName = what + '_' + target + '_' + feat
```

Comment:
No need to change, but less chance for error if this is constructed as `'_'.join([what,target,feat])`.  This happens a couple times below, as well.

Reply:
fixed.